### DISCUSSION TITLE
New version without need for MATLAB?

### DISCUSSION THREAD

**Discussion by wharfbanger** [alexisInSAR/EZ-InSAR#98] [Ideas and further developments]
Hello, I note in the documentation from 2023 a new version of EZ-InSAR was coming soon.  The new version would be written in Python to avoid the need for  MATLAB.  Just wondering if this is still happening?

**Comment by geoenviron98** [comment]
I think because this software uses StaMPS processor, so that's way it requires MATLAB

**Comment by alexisInSAR** [comment]
Hi, 

Unfortunately, MATLAB will still be needed for StaMPS.  

I maintain EZ-InSAR when I have time because the project has been finished for 1 year.

Of course, our plan is to offer version 3 of EZ-InSAR in Python (see the EZ-InSAR publication). I am trying to work on it but it will take months to write this version because it is not just a translation but a restructuring of the whole toolkit. I cannot promise anything and the Python version will be available when it is ready 😇.  

Kind regards, 

Alexis
